images-dist/modules/kogito-maven/common/module.yaml

```diff
@@ -23,16 +23,20 @@ version: "main"
 envs:
   - name: "MAVEN_VERSION"
     description: "The Maven version to setup with this module"
+    value: "3.9.6"
   - name: "MAVEN_HOME"
     value: "/usr/share/maven"
   - name: "MAVEN_CMD"
-    value: "/usr/share/maven/bin/mvnw"
+    value: "/usr/share/maven/bin/mvn"
   - name: "MAVEN_SETTINGS_PATH"
     description: "The location of the settings.xml file"
     value: "${KOGITO_HOME}/.m2/settings.xml"
   - name: "MAVEN_CONTAINER_BUILD_SETTINGS_PATH"
     description: "The location of the settings.xml file during container image build"
     value: "${KOGITO_HOME}/.m2/maven-m2-repo-via-http-settings.xml"
+  - name: "ALTERNATIVE_MAVEN_HOME"
+    description: " (Temporary Fix) Path to the alternative Maven installation until a new base image with Maven 3.9 is available."
+    value: "/usr/share/apache-maven-3.9.6"
```

Comment:
now it is hardcoded?

Reply:
The changes are coming from this [PR](https://github.com/apache/incubator-kie-tools/pull/3167/files) in upstream

Reply:
We will remove it once UBI9 JDK 17 is releaed with Maven 3.9: https://issues.redhat.com/browse/SRVLOGIC-212